src/developer.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+//Developer start
+
+console.log("class Developer");
+export const developer = class Developer {
```

Comment:
```suggestion
export class Developer {
```
так лучше. Сразу класс экспортируем и оставляем в названии написание с заглавной буквы, чтобы те, кто далее будет наш класс импортировать, знали, что это действительно класс